docs/installation.md

```diff
@@ -0,0 +1,118 @@
+# Installation
+
+Arlon CLI downloads are provided on GitHub. The CLI is not a self-contained standalone executable though.
+It is required to point the CLI to a management cluster and set up the Arlon controller in this management cluster.
+
+For a quickstart minimal demonstration setup, follow the instructions to set up a KIND based testbed with Arlon and ArgoCD running  [here](https://github.com/arlonproj/arlon/blob/main/testing/README.md).
+
+Please follow the manual instructions in [this](#customised-setup) section for a customised setup or refer the instructions for automated installation [here](#automatic-setup).
```

Comment:
automated installation is a v0.10+ feature only

Reply:
Cleaned up the section. Thanks.